classes/OpenCFP/Provider/RouteServiceProvider.php

```diff
@@ -70,6 +70,10 @@ public function register(Application $app)
         // Admin::Review
         $secureRoutes[] = $app->get('/admin/review', 'OpenCFP\Http\Controller\Admin\ReviewController::indexAction')->bind('admin_reviews');
 
+        // API end points
```

Comment:
Thoughts on following?

| method | route | description |
| --- | --- | --- |
| `POST` | `/api/talks` | Given JSON payload representing a talk, creates talk for authenticated user and issues a 201 Created upon success, appropriate error otherwise |
| `GET` | `/api/talks` | Returns JSON collection of all talks for authenticated user |
| `GET` | `/api/talks/{id}` | Returns a particular talk for authenticated user. Returns appropriate responses for unauthorized or non-existent talks |
| `DELETE` | `/api/talks/{id}` | Removes a talk. |

This structure would allow functionality that would support a feature for "import from another OpenCFP instance" (in response to things like https://twitter.com/ocramius/status/554718836134780929) whereby user could simply provide a link to the root of CFP site and execution could take it from there. 

That said, adding something to `GET /api` would be a good idea to identify the API root (and the fact that the API is enabled or not in the case you make the API able to be toggled by configuration)